deployer/deployer.py

```diff
@@ -506,3 +518,110 @@ def deploy(self, environment='pre', eureka=True, health_check=None, health_check
 
         if environment != 'pro':
             self.set_scheduler(version)
+
+    def deploy_elb(self, health_check='healthcheck', health_check_port=8000):
+        self.elbs = [self.app]
+
+        elb_data = self.get_or_create_loadbalancer_data(health_check, health_check_port)
+
+        self.validate_loadbalancer(elb_data, health_check, health_check_port)
+
+        if self.elb_dns:
+            domain_name = elb_data["loadBalancer"]["DNSName"]
+            self.check_or_create_route53(domain_name)
+
+    def get_or_create_loadbalancer_data(self, health_check, health_check_port):
+        elb_data = self.get_loadbalancer_data()
+
+        if elb_data is None:
+            if not self.create_loadbalancer(health_check, health_check_port):
+                print("load balancer creation failed", file=sys.stderr)
+                exit(1)
+            elb_data = self.get_loadbalancer_data()
+
+        return elb_data
+
+    def validate_loadbalancer(self, elb_data, health_check, health_check_port):
```

Comment:
@padilo this method is static, remove the slef parameter and add "@staticmethod" anotation